gateway/rpc_storage_handler.go

```diff
@@ -102,6 +102,7 @@ const (
 	OauthClientAdded        string = "OauthClientAdded"
 	OauthClientRemoved      string = "OauthClientRemoved"
 	OauthClientUpdated      string = "OauthClientUpdated"
+	deleteAPICache          string = "DeleteAPICache"
```

Comment:
With `NoticeAPIDeleteCache`, this seems redundant (same value in two vars).

Reply:
both are used in two different contexts, one from redis and other from rpc

Reply:
they have equality, may as well just keep the strong typed value around